### ISSUE TITLE
Bring SRID in from_postgis()

### ISSUE THREAD

**Issue by adamboche** [geopandas/geopandas#451]
I have a geometry in postgis and query it with `df = gpd.GeoDataFrame.from_postgis()`. I then want to use `df.to_crs()`, but `df` doesn't seem to have a crs already, so it says `
ValueError: Cannot transform naive geometries.  Please set a crs on the object first.` 

I can set `df.crs = ...`, but that seems fragile. Is it possible to make geopandas get the `srid` from postgis when it makes the query, and automatically set the crs?

**Comment by kuanb** [comment]
I'd like to second this. In my workflows, I tend to have a `parse_wkt` function that performs the following step:

```
    srid = None
    if geo_data.startswith('SRID'):
        semi_idx = geo_data.index(';')
        srid = int(geo_data[5:semi_idx])
        geo_data = geo_data[semi_idx + 1:]

    geo = shapely.wkt.loads(geo_data)
```

The `srid` is then held and passed along downstream to create a `crs` dictionary. Seeing as this logic is repeated consistently, and a common use case is to manipulate spatial query results from PG in memory, this would be a useful addition.

**Comment by emiliom** [comment]
Yeah, I ran into this last November, with the [current release](https://github.com/geopandas/geopandas/releases/tag/v0.2.1).

Note that you can be pre-emptive and pass the `crs` as an argument to the `from_postgis` call:

```python
seas = GeoDataFrame.from_postgis("select * from world_seas", conn, 
                                 geom_col='geometry', crs={'init': 'epsg:4326'}, 
                                 coerce_float=False)
```
But that's also fragile.

It would also be nice if geopandas could guess the `geom_col`; if only one `geometry`-type column is pulled by the query, it should be unambiguous.

I may be able to look into this and provide a PR over the next month or so, specially for the crs/SRID part, and specially if I can get some pointers from the developers. So far I see the relevant code (`read_postgis`) is at https://github.com/geopandas/geopandas/blob/master/geopandas/io/sql.py#L9, and to a lesser extent at https://github.com/geopandas/geopandas/blob/master/geopandas/geodataframe.py#L195 (the `from_postgis` warapper to `read_postgis`).

**Comment by emiliom** [comment]
This need was noted as far back as August 2014, in #161; linking here to that issue b/c there are additional details and discussions there that may be helpful when trying to add this capability.

**Comment by jdmcbr** [comment]
Closed by #627.
